keepassxc-browser/content/autocomplete.js

```diff
@@ -1,10 +1,21 @@
 'use strict';
 
 var kpxcAutocomplete = {};
+kpxcAutocomplete.autoSubmit = false;
 kpxcAutocomplete.elements = [];
+kpxcAutocomplete.started = false;
+kpxcAutocomplete.index = -1;
+kpxcAutocomplete.input = undefined;
 
 kpxcAutocomplete.create = function(input, showListInstantly = false, autoSubmit = false) {
-    let _index = -1;
+    //let _index = -1;
+    console.log('start');
+
+    kpxcAutocomplete.autoSubmit = autoSubmit;
+    kpxcAutocomplete.input = input;
+    if (!kpxcAutocomplete.started) {
+        kpxcAutocomplete.started = true;
```

Comment:
No need to check if it is false if you are going to set it to true no matter what